python/ml_croissant/ml_croissant/_src/operation_graph/operations/extract.py

```diff
@@ -44,9 +63,14 @@ def __call__(self):
                     included_files.append(epath.Path(basepath) / file)
         # We need to sort `files` to have a deterministic/reproducible order.
         included_files.sort()
+        fullpaths = [
```

Comment:
NIT: Given that this is a long list comprehension, you might want to add a comment to increase readibility (e.g. why is "[1:]" needed?)

Reply:
Done.